### DISCUSSION TITLE
How can i get a blockpalette from a Batch?

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#998] [Help] [answered]
Hello! :D

I need a blockpalette from a RelativeBlockBatch

how can i do this? thanks :D

**Comment by MrGazdag** [comment]
I don't quite understand what you mean, RelativeBlockBatches don't have a palette, they use `Block` references to keep track of block changes. You can look at the source code of [RelativeBlockBatch](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/instance/batch/RelativeBlockBatch.java) to see how they work.

**Reply by DasLixou** [reply]
O ok. I have a system that reads a file into multiple batches, and when chunks get loaded, it can just paste the batch. But for saving, I need the block palette of the batch. Is there a way to get it?

**Comment by DasLixou** [comment, marked as answer]
Ok i just dont use BlockBatches anymore
